<launch>
  <arg name="config_file"/>
  <arg name="robot_id" default="robot0"/>

  <node pkg="genmos_object_search_ros2"
        name="run_search"
        exec="run_search.py">
    <param name="config_file" value="$(var config_file)"/>
    <param name="robot_id" value="$(var robot_id)"/>
    <param name="world_frame" value="graphnav_map"/>
    <remap from="~/search_region_cloud_3d" to="/local_cloud_publisher/region_points"/>
    <remap from="~/search_region_cloud_2d" to="/local_cloud_publisher/region_points"/>
    <remap from="~/search_region_center" to="/simple_sim_env/robot_pose"/>
    <remap from="~/robot_pose" to="/simple_sim_env/robot_pose"/>
    <remap from="~/object_detections" to=""/>
    <remap from="~/object_detections" to=""/>
  </node>
</launch>
